### PR TITLE
Template for build or installation type of issue

### PR DESCRIPTION
**Pull request by Harshini-Gadige**
Template for build or installation type of issue

### CODE REVIEW

File: .github/30-build-installation-issue.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Build/Installation Issue
+about: Use this template for build/installation issues
+
+---
+
+<em>tag:build_template</em>
```
Comment (nfelt) [inline .github/30-build-installation-issue.md]: Does this tag do anything in particular?  Is it for tracking purposes?

File: .github/30-build-installation-issue.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+**Please provide the following information**
+
+**System information**
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04) :
+- TensorBoard version :
+- TensorFlow version(if different from TensorBoard) :
```
Comment (nfelt) [inline .github/30-build-installation-issue.md]: nit: space between "version" and "(if"

Also a nit, but no spaces before the colons on this line and the two above

File: .github/30-build-installation-issue.md (hunk 1 above)
Comment (hgadig) [reply .github/30-build-installation-issue.md]: Yes that is for our dashboard to pull the metrics.

File: .github/30-build-installation-issue.md (hunk 2 above)
Comment (hgadig) [reply .github/30-build-installation-issue.md]: Done

**Comment by nfelt** [comment]
@hgadig If it's not too much trouble, would you mind folding the other PRs #1939 #1940 #1941 into this PR so that it's easier to review all the templates together?

File: .github/20-feature-request.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+**Describe the feature and the current behavior/state.**
+
+**Will this change the existing plugins? If so, please provide plugin name and details**.
+
+**Is this about adding a new plugin?**
+
+**Who will benefit with this feature?**
+
+**Is there any existing workaround for this ?**
```
Comment (stephanwlee) [inline .github/20-feature-request.md]: Sorry for nit: can you remove the whitespace before the question mark?

File: .github/20-feature-request.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version (you are using) :
+- Are you willing to contribute it (Yes/No) :
+
+
+
+**Describe the feature and the current behavior/state.**
```
Comment (stephanwlee) [inline .github/20-feature-request.md]: To make it consistent with other files, please end it with ":" or change those files to end with the period.

File: .github/00-bug-performance-issue.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+tag:bug_template</em>
```
Comment (nfelt) [inline .github/00-bug-performance-issue.md]: Missing the leading `<em>` tag

File: .github/00-bug-performance-issue.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version(from pip package, also printed out when running `tensorboard`) :
```
Comment (nfelt) [inline .github/00-bug-performance-issue.md]: nit, space between `version` and `(from`

Comment (nfelt) [inline .github/00-bug-performance-issue.md]: nit: can we avoid spaces before the trailing colons, on all lines in all templates?

Comment (nfelt) [inline .github/00-bug-performance-issue.md]: nit: first word should be capitalized `TensorBoard` with a capital B

File: .github/10-documentation-issue.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+---
+
+<em>tag:doc_template</em>
+
+**Please provide the following information**
+
+**System information**
+- TensorFlow version :
+- Tensorboard version :
```
Comment (nfelt) [inline .github/10-documentation-issue.md]: nit: first word should be capitalized `TensorBoard` with a capital B

File: .github/20-feature-request.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+about: Use this template for raising a feature request
+
+---
+
+<em>tag:feature_template</em>
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version (you are using) :
```
Comment (nfelt) [inline .github/20-feature-request.md]: nit: first word should be capitalized `TensorBoard` with a capital B

File: .github/30-build-installation-issue.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+**Please provide the following information**
+
+**System information**
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- TensorBoard version:
+- TensorFlow version (if different from TensorBoard):
+- Python version:
+- Installed using virtualenv? pip? conda?:
+- Bazel version (if compiling from source):
+- GCC/Compiler version (if compiling from source):
```
Comment (nfelt) [inline .github/30-build-installation-issue.md]: I don't think we need to ask for these last three fields (compiler, cuda, gpu); they're not really relevant to TensorBoard.

File: .github/10-documentation-issue.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+- Doc Link :
+
+**Describe the documentation issue**
+
+**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**
```
Comment (nfelt) [inline .github/10-documentation-issue.md]: nit: `Issue` should be lowercase

File: .github/20-feature-request.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+- Are you willing to contribute it (Yes/No) :
```
Comment (nfelt) [inline .github/20-feature-request.md]: This isn't really "System information" - could it be included in the questions below?

File: .github/20-feature-request.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+**Will this change the existing plugins? If so, please provide plugin name and details**.
+
+**Is this about adding a new plugin?**
+
+**Who will benefit with this feature?**
+
+**Is there any existing workaround for this ?**
+
+**Any Other info.**
```
Comment (nfelt) [inline .github/20-feature-request.md]: nit: `Other` should be lowercase

File: .github/00-bug-performance-issue.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+- TensorFlow version if different from TensorBoard :
+- OS Platform and version (e.g., Linux Ubuntu 16.04) :
+- Python version (e.g. 2.7, 3.5) :
```
Comment (stephanwlee) [inline .github/00-bug-performance-issue.md]: I think this was there before but would you mind cleaning this up and make `e.g.` consistent within/across files? I believe `e.g.,` is the correct phrase.

File: .github/20-feature-request.md (hunk 3 above)
Comment (hgadig) [reply .github/20-feature-request.md]: Done

File: .github/20-feature-request.md (hunk 4 above)
Comment (hgadig) [reply .github/20-feature-request.md]: Done

File: .github/10-documentation-issue.md (hunk 7 above)
Comment (hgadig) [reply .github/10-documentation-issue.md]: Done

File: .github/00-bug-performance-issue.md (hunk 6 above)
Comment (hgadig) [reply .github/00-bug-performance-issue.md]: Done

File: .github/00-bug-performance-issue.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+</em>tag:bug_template</em>
```
Comment (stephanwlee) [inline .github/00-bug-performance-issue.md]: It should be 
```suggestion
<em>tag:bug_template</em>
```

File: .github/20-feature-request.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+about: Use this template for raising a feature request
+
+---
+
+<em>tag:feature_template</em>
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version (you are using):
```
Comment (stephanwlee) [inline .github/20-feature-request.md]:
```suggestion
- TensorBoard version (you are using):
```

File: .github/10-documentation-issue.md [hunk 16]
```diff
@@ -0,0 +1,18 @@
...
+about: Use this template for documentation related issues
+
+---
+
+<em>tag:doc_template</em>
+
+**Please provide the following information**
+
+**System information**
+- TensorFlow version :
```
Comment (stephanwlee) [inline .github/10-documentation-issue.md]: Can we remove the whitespace before colon?

File: .github/10-documentation-issue.md (hunk 7 above)
Comment (stephanwlee) [reply .github/10-documentation-issue.md]: I think you are missing a commit

**Comment by hgadig** [comment]
> Almost done! Please in the future fork the repo and use branch in your repo so, instead of nitpicking many times, we can just send a PR to your branch. Thanks!

Sure.

**Comment by hgadig** [comment]
I will update all the comments related to nit etc. I will work on this sometime next week as I've got other things to do.

**Comment by stephanwlee** [comment]
I believe this Pull Request is now mostly obsolete given https://github.com/tensorflow/tensorboard/pull/2178. Please reopen (or create another PR) if you think we need more improvements.

**Pull request closed without merging**